web/src/lib/components/forms/create-user-form.svelte

```diff
@@ -69,11 +69,11 @@
           error = 'Error create user account';
           isCreatingUser = false;
         }
-      } catch (e) {
+      } catch (error_) {
```

Comment:
there's quite a few places throughout with trailing underscore, which seems a bit funny to me. I thought I'd point this out in case it was done by the automatic fixer and we thought it was worth changing. i'm not sure how to really identify all the places it did it though. this PR almost crashes my browser as is :laughing:

Reply:
lol. Yeah, maybe this PR is a little much as-is. At least all the lints are fixed now. Thanks for noticing the underlines.